src/components/SummaryManagement/SummaryReceiverPopup.jsx

```diff
@@ -0,0 +1,113 @@
+/* eslint-disable no-underscore-dangle */
+/* eslint-disable react/prop-types */
+import React, { useState, useEffect } from 'react';
+import { Button, Modal, ModalHeader, ModalBody, ModalFooter, Container, Alert } from 'reactstrap';
+import MembersAutoComplete from './MembersAutoComplete';
+//import hasPermission from 'utils/permissions';
+
+const TeamMembersPopup = React.memo(props => {
+  // debugger;
+  const closePopup = () => {
+    props.onClose();
+  };
+  const [selectedUser, onSelectUser] = useState(undefined);
+  const [isValidUser, onValidation] = useState(true);
+  const [searchText, setSearchText] = useState('');
+
+  const onAddUser = () => {
+    cd;
+    if (
+      selectedUser &&
+      (!props.members ||
+        !props.members.teamMembers ||
+        !props.members.teamMembers.some(x => x._id === selectedUser._id))
+    ) {
+      props.onAddUser(selectedUser, props.selectedSummaryGroupId);
+      setSearchText('');
+    } else {
+      onValidation(false);
+    }
+  };
+  const selectUser = user => {
+    onSelectUser(user);
+    onValidation(true);
+  };
+
+  useEffect(() => {
+    onValidation(true);
+  }, [props.open]);
+
+  return (
+    <Container fluid>
+      <Modal isOpen={props.open} toggle={closePopup}>
+        <ModalHeader
+          toggle={closePopup}
+        >{`Members of ${props.selectedSummaryGroupName}`}</ModalHeader>
```

Comment:
Label the popups so you can differentiate between the member and receiver popup
```suggestion
        >{`Receivers of ${props.selectedSummaryGroupName}`}</ModalHeader>
```

Reply:
update please check, thanks